.aws/src/config/index.ts

```diff
@@ -43,6 +43,9 @@ const prefect = {
       'GCE_KEY',
       'BRAZE_API_KEY',
       'BRAZE_REST_ENDPOINT',
+      'AURORA_DB_USER',
+      'AURORA_DB_PASSWORD',
+      'AURORA_DB_HOST',
```

Comment:
Could you add these to the ReadMe under 'Initial Deployment'? There's a [running list parameters](https://github.com/Pocket/data-flows#2-aws-ssm-parameter-store) we need to take if we'd ever need to deploy this from scratch.

Reply:
Yeah if we have secrets to production resources in Dev we should delete them and replace them with dummy values. The dev AWS account is not a safe place for this information.

Reply:
Thanks for the call-out. I've removed Dbt and Snowflake credentials from Pocket-Dev.